portfolio/src/main/webapp/gallery.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Gallery | My Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+  </head>
+  <body>
+    <div id="content">
+      <div class="header-div">
+        <h1 class="heading-text">Gallery</h1>
+      </div>
+      <div class="gallery-row">
+        <div class="gallery-col">
+          <img class="gallery-img" src="/images/gallery-images/bhc-fried-chicken.jpg" alt="Fried chicken">
+          <img class="gallery-img" src="/images/gallery-images/frangipani.jpg" alt="Frangipani">
+          <img class="gallery-img" src="/images/gallery-images/cupcakes.jpg" alt="Cupcakes">
+          <img class="gallery-img" src="/images/gallery-images/christmas-in-shanghai.jpg" alt="Christmas tree">
+          <img class="gallery-img" src="/images/gallery-images/tianzifang.jpg" alt="Narrow street in Shanghai">
+        </div>
+        <div class="gallery-col">
+          <img class="gallery-img" src="/images/gallery-images/the-bund.jpg" alt="View of city-scape">
+          <img class="gallery-img" src="/images/gallery-images/hashcafe.jpg" alt="Croissants">
+          <img class="gallery-img" src="/images/gallery-images/namsan-tower.jpg" alt="Namsan Tower">
+          <img class="gallery-img" src="/images/gallery-images/sulbing-bingsu.jpg" alt="Bingsu (Shaved Ice)">
+          <img class="gallery-img" src="/images/gallery-images/innisfree-cafe.jpg" alt="Fluffy pancakes.">
+        </div>
+      </div>
+      <div id="contacts-div" class="flex-container">
+        <div class="flex-container">
+          <span class="material-icons">code</span>
+          <a href=https://github.com/anna-giang/ target="_blank"><p class="body-text">GitHub</p></a>
+        </div>
+        <div class="flex-container">
+          <span class="material-icons">chat</span>
+          <a href="https://linkedin.com/in/anna-giang1999" target="_blank"><p class="body-text">LinkedIn</p></a>
+        </div>
+        <div class="flex-container">
+          <span class="material-icons">mail_outline</span>
+          <a href="mailto:anna.giang1999@gmail.com" target="_blank"><p class="body-text">Email</p></a>
+        </div>
+      </div>
+    </div>
+  </body>
+</html>
```

Comment:
nit: github is suggesting that this file doesn't have a newline. It's a bit hard for me to see the line endings of files via github, so just double checking.

Reply:
It should have a blank line now

Reply:
Yep, seems to be there now. Thanks :)